tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md

```diff
@@ -0,0 +1,111 @@
+---
+title: TiDB Cloud TPC-C Performance Test Report
+summary: Introduce the TPC-C performance test results for TiDB Cloud.
+---
+
+# TiDB Cloud TPC-C Performance Test Report
+
+This document provides the TPC-C performance test steps and results for a TiDB Dedicated cluster with the TiDB version of v7.1.0. This report can also be used as a reference for the performance of TiDB v7.1.0 clusters.
+
+## Test overview
+
+This test aims at showing the TPC-C performance of TiDB v7.1.0 in the Online Transactional Processing (OLTP) scenario.
+
+## Test environment
+
+### TiDB cluster
+
+The test is conducted on a TiDB cluster with the following settings:
+
+- Cluster type: [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier)
+- Cluster version: v7.1.0
+- Cloud provider: AWS (us-west-2)
+- Cluster configuration:
+
+   | Node type | Node size | Node quantity | Node storage |
+   |:----------|:----------|:----------|:----------|
+   | TiDB      | 16 vCPU, 32 GiB | 2 | N/A |
+   | TiKV      | 16 vCPU, 64 GiB | 3 | 1000 GiB |
+
+### Benchmark executor
```

Comment:
Please also add the AWS region for the Benchmark executor.

Reply:
Done https://github.com/pingcap/docs/pull/13696/commits/5afd391e501e127a52a415db6fad6738291c8a7d